C++/p004-Layers-and-Object.cpp

```diff
@@ -1,51 +1,53 @@
 /**
- * Creates a dense layer 
- * 
+ * Creates a dense layer
+ *
  * Associated tutorial https://www.youtube.com/watch?v=TEWy9vZcxW4
  */
 #include <iostream>
 #include <random>
 #include <vector>
+#include <algorithm>
+#include <functional>
+
 using namespace std;
 
-/* A helper function to get a random float in range [low, high] */
-double getRandomDouble(double low, double high) {
```

Comment:
generator is destroyed each time the function returns. It causes to produce the same random number. This can be solved by making the `gen` object public